core/src/main/java/io/grpc/internal/ChannelExecutor.java

```diff
@@ -47,7 +49,7 @@
  * order as they are submitted.
  */
 @ThreadSafe
-final class ChannelExecutor {
+public final class ChannelExecutor {
```

Comment:
Reverted `ChannelExecutor`'s visibility. Switched to using `SerializingExecutor` because it provides the same semantics with `ChannelExecutor` except that it doesn't provide delayed draining which my test doesn't need.